storybookjs/storybook
# RN app: upgrade ws from 3.3.3 to 4.1.0

Fixes #2658 

## How to test
Launch CRNA example

We don't seem to need any migration, see https://github.com/websockets/ws/issues/1256#issuecomment-368229506
